src/main/java/seedu/address/ui/ClientListPanel.java

```diff
@@ -32,9 +32,20 @@ public ClientListPanel(ObservableList<Client> clientList) {
         super(FXML);
         clientListView.setItems(clientList);
         clientListView.setCellFactory(listView -> new PersonListViewCell());
-        numClients.setText(Integer.toString(clientList.size()) + " Records");
+        numClients.setText(numClientsString(clientList));
         clientList.addListener((ListChangeListener<? super Client>)
-                c -> numClients.setText(Integer.toString(clientList.size()) + " Records"));
+                c -> numClients.setText(numClientsString(clientList)));
+    }
+
+    /**
+     * Returns a string denoting the number of records for {@code Client} currently shown in the {@code clientList}.
+     */
+    private String numClientsString(ObservableList<Client> clientList) {
```

Comment:
was thinking about making this a generic name, and then add into UiPart as an abstracted method also

Reply:
hmm, this method does not belong to other Ui elements though